### PR TITLE
Issue #43, fix install script.

### PR DESCRIPTION
**Pull request by gled-rs**
Fix the installation script, mainly some django issue, django 1.9 being pulled as a dependency instead of django 1.7 that we need.

### CODE REVIEW

File: requirements/django.txt [hunk 1]
```diff
@@ -8,8 +8,8 @@ django-jsonfield==0.9.13
 django-countries==2.1.2
 git+http://github.com/cdr-stats/django-pagination.git
 django-admin-tools==0.7.2
+django-dajaxice-ng==0.5.6.4
```
Comment (areski) [inline requirements/django.txt]: Any reasons why you downgraded django-dajaxice-ng to that version?

Comment (gled-rs) [inline requirements/django.txt]: Yes, the version you were using was pulling django 1.9 as a dependency.

Comment (areski) [inline requirements/django.txt]: Ok I found it https://github.com/ifanrx/django-dajaxice/blob/master/setup.py
This is still annoying to downgrade so much

Comment (gled-rs) [inline requirements/django.txt]: we could maybe stick with 0.7.0.2, seems like the change was introduced at that time.

**Comment by areski** [comment]
Great fix, I merged and tested https://github.com/cdr-stats/cdr-stats/commit/e9a8313430e9fd4b16df71bf0f72c5ae4c4660eb

**Comment by areski** [comment]
also v3.1.3 is released

**Pull request closed without merging**